todo_app/templates/index.html

```diff
@@ -25,73 +27,81 @@ <h3 class="border-bottom border-gray pb-2 mb-0">Add Item</h3>
-                  Expand
-                </button>
-                <button class="btn btn-secondary btn-sm dropdown-toggle" type="button" id="dropdownMenuButton_{{card['id']}}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                  Actions
-                </button>
-                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton_{{card['id']}}">
-                  {% for move_list in view_model.lists %}
-                    {% if move_list['id'] != card['idList'] %}
-                      <a class="dropdown-item" href="/move/{{card['id']}}/{{move_list['id']}}">Move to: {{move_list['name']}}</a>
-                    {% endif %}
-                  {% endfor %}
-                  <div class="dropdown-divider"></div>
-                  <a class="dropdown-item" href="/remove/{{card['id']}}">Remove</a>
-                </div>
-              </div>
+    <div class="media text-muted pt-3">
+      <div class="media-body pb-3 mb-0 small lh-125 border-bottom border-gray">
+        <div class="d-flex justify-content-between align-items-center w-100">
+          <h6 class="text-gray-dark" id="doing_item_{{card['id']}}">{{card['name']}}</h6>
+          <div class="dropdown">
+            <button class="btn btn-secondary btn-sm" type="button" data-toggle="collapse" data-target="#collapse{{card['id']}}" aria-expanded="false" aria-controls="collapse{{card['id']}}">
+              Expand
+            </button>
+            <button class="btn btn-secondary btn-sm dropdown-toggle" type="button" id="dropdownMenuButton_{{card['id']}}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              Actions
+            </button>
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton_{{card['id']}}">
+              {% for move_list in view_model.lists %}
+                {% if move_list['id'] != card['idList'] %}
+                  <a class="dropdown-item" href="/move/{{card['id']}}/{{move_list['id']}}">Move to: {{move_list['name']}}</a>
+                {% endif %}
+              {% endfor %}
+              <div class="dropdown-divider"></div>
+              <a class="dropdown-item" href="/remove/{{card['id']}}">Remove</a>
+            </div>
           </div>
-          <div class="collapse" id="collapse{{card['id']}}">
+        </div>
+        <div class="collapse" id="collapse{{card['id']}}">
+          {% if card['desc'] %}
             <small><b>Description:</b> {{card['desc']}}</small>
-            {% if card['due'] %}
-              <br>
-              <small><b>Due:</b> {{card['due'].strftime('%d/%m/%Y')}}
-            {% endif %}
-          </div>
+          {% endif %}
+          {% if card['desc'] and card['due'] %}
+            <br>
+          {% endif %}
+          {% if card['due'] %}
+            <small><b>Due:</b> {{card['due'].strftime('%d/%m/%Y')}}</small>
+          {% endif %}
```

Comment:
You have this same block multiple times in the file and then have to update it in multiple places whenever you want a change.
You could instead put it in e.g a `card.html` file and reference it with `{% include 'card.html' %}` in several places in this file.